src/standalone/userJupyterServer/userServerUrlProvider.ts

```diff
@@ -77,39 +73,37 @@ const GlobalStateUserAllowsInsecureConnections = 'DataScienceAllowInsecureConnec
 
 @injectable()
 export class UserJupyterServerUrlProvider
-    implements
-        IExtensionSyncActivationService,
-        IDisposable,
-        IInternalJupyterUriProvider,
-        JupyterServerProvider,
-        JupyterServerCommandProvider
+    extends Disposables
+    implements IExtensionSyncActivationService, IDisposable, JupyterServerProvider, JupyterServerCommandProvider
```

Comment:
Temporary hacks to support multipllle interfaces as we support both old an new APIs, once we remove the old API, this class will get much smaller